backend/database/database.go

```diff
@@ -391,6 +394,43 @@ func (d *Database) ReviewsByVendorID(vendorID uuid.UUID) ([]Review, error) {
 	return result, rows.Err()
 }
 
+//query to find out if the vendor has been starred, it returns true if it is false if not
+func (d *Database) StarredVendorFavorite(VendorID int, ReviewsID int) (bool, error) {
+	var starred bool
+
+	if err := d.db.QueryRow("SELECT (ID >= ?) FROM reviews WHERE VendorID = ?",
```

Comment:
You cannot have a filter in `SELECT`. You probably meant to put `(ID >= ?)` as a `WHERE` condition. Also, the `VendorID` is defined as a `CHAR(36)` but you are passing an integer.